knimin/handlers/pm_plate_list.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+from tornado.web import authenticated
+from knimin.handlers.base import BaseHandler
+from knimin import db
+from knimin.handlers.access_decorators import set_access
+
+
+@set_access(['Admin'])
```

Comment:
I think we need to create a different access level for the Plate Mapper and not use Admin. However, no need to change now, and I added it to the TODO list.